### DISCUSSION TITLE
checking for changes other than baseBranch

### DISCUSSION THREAD

**Discussion by wdfinch** [vercel/turborepo#239] [Help] [answered]
Hi there, so as I understand it, this tool will check for changes between the `baseBranch` option and the current `git` branch to see if changes are present. Is there any other way to detect changes? In my workflow, I don't merge to origin master immediately and instead use release branches. Therefore, my master branch would almost always show changes between my current branch and this wouldn't help me much.

Are there any other methods for detecting changes? For example, all I would need is for the tool to compare a hash of the current files in a workspace against the hash for the last cache on disk. If they differ, the cache is stale.

**Comment by jaredpalmer** [comment, marked as answer]
You can use the `--since` flag (https://turborepo.org/docs/reference/command-line-reference#--since)

**Reply by wdfinch** [reply]
Thanks!
